examples/fastspeech2_multispeaker/conf/fastspeech2.v1.yaml

```diff
@@ -0,0 +1,74 @@
+# This is the hyperparameter configuration file for FastSpeech2 v1.
+# Please make sure this is adjusted for the LJSpeech dataset. If you want to
+# apply to the other dataset, you might need to carefully change some parameters.
+# This configuration performs 200k iters but a best checkpoint is around 150k iters.
+
+###########################################################
+#                FEATURE EXTRACTION SETTING               #
+###########################################################
+hop_size: 221            # Hop size.
+format: "npy"
+
+
+###########################################################
+#              NETWORK ARCHITECTURE SETTING               #
+###########################################################
+fastspeech_params:
+    n_speakers: 5
```

Comment:
5 ?

Reply:
Any value of user dataset its just base config file